src/main/java/de/tum/in/www1/artemis/web/rest/UserResource.java

```diff
@@ -115,6 +126,55 @@ public ResponseEntity<Void> updateUserNotificationDate() {
         return ResponseEntity.ok().build();
     }
 
+    /**
+     * PUT users/profile-picture : upload a profile picture
+     *
+     * @param file the image file that is being uploaded
+     * @return the ResponseEntity with status 200 (OK) and with body of current user
+     */
+    @PutMapping("users/profile-picture")
+    @EnforceAtLeastStudent
+    public ResponseEntity<UserDTO> updateProfilePicture(@RequestPart MultipartFile file) throws URISyntaxException {
```

Comment:
I guess it's not entirely clear where the actual difference is, but most other settings use the `account/....` endpoint. I think also doing that for the profile picture, might align a bit better with the rest of the settings endpoints.

Reply:
Hey! Yeah, you are right, in my mind the account api was used for mainly managing credentials (username, password, tokens) and the user api was used for metadata. I was a bit on the fence when deciding which one to go for. The decision was mainly influenced by the administration api also being called "admin/users". But I'll change it to /account, I think that makes more sense. 👍🏼

Reply:
I changed the api route to /api/account/profile-picture and moved the functions to the AccountResource in this [commit](https://github.com/ls1intum/Artemis/pull/9277/commits/d9a77eb44ab7f0ade78c98d3fbb56c58b9c85ad5). 👍🏼